types/Cache/Repository.php

```diff
@@ -18,12 +18,12 @@
 assertType('mixed', $cache->pull('cache', function (): int {
     return 30;
 }));
-assertType('int', $cache->sear('cache', function (): int {
+assertType('33', $cache->sear('cache', function (): int {
     return 33;
 }));
-assertType('int', $cache->remember('cache', now(), function (): int {
+assertType('36', $cache->remember('cache', now(), function (): int {
     return 36;
 }));
```

Comment:
I would assume that a callback with `int` return type would result in a type called `int` and not `36`. There seems to be a problem elsewhere.

All these type assertions should remain unchanged.

Reply:
I think that since PHPStan 2.0, types are inferred more narrowly, in this case as 36 literal type.

Reply:
I hope that's something that can be configured somewhere, since the returned type is a string with the number, so the "literal" part is misleading.